src/lib/viewers/doc/DocLoader.js

```diff
@@ -22,6 +23,12 @@ const VIEWERS = [
         REP: 'pdf',
         EXT: DOCUMENT_EXTENSIONS
     },
+    {
+        NAME: 'SinglePage',
```

Comment:
The current behavior is that if we disable the Document viewer, we fallback to the Presentation viewer. Can you put this below the next Presentation viewer entry? This means that for your viewer to take effect, you would have to disable both the Document and Presentation viewer.

We can add this to the Readme.md file in the doc viewer folder.

Reply:
Did I rearrange this correctly I did not completely understand how this works?

Reply:
Yep, this looks good to me. We pick the first viewer in this list that
1. Is enabled
2. Can support the passed in file extension (EXT)
3. Can support a representation that is provided by the API (REP)

Reply:
Hey thanks for that, clears it up I think I was getting stuck on the REP part I didn't completely understand that.